packages/fuels-test-helpers/src/utils.rs

```diff
@@ -128,8 +135,8 @@ mod tests {
         }
 
         #[tokio::test]
-        async fn returns_value_on_success() -> anyhow::Result<()> {
-            let successfully_generates_value = || async { Ok(12345u64) as anyhow::Result<u64> };
+        async fn returns_value_on_success() -> Result<()> {
+            let successfully_generates_value = || async { Ok(12345u64) as Result<u64> };
 
             let value = retry(
```

Comment:
We might as well remove the `retry` helper. 

Made sense when we didn't have the client API for tweaking the block height, now I'm not sure anybody uses it.

The only place it is 'used' is `confirm_blocks_created` which probably nobody uses and probably shouldn't use since we now have a reliable API to tweak the block height.

Reply:
Agreed